acceptance_tests/features/create_message_internal.feature

```diff
@@ -0,0 +1,44 @@
+Feature: Internal user to send message
+  As an internal ONS user
+  I need to be able to send a secure message
+  So that I can query data that has been received
+
+  Background: User already logged in
+    Given the internal user is already signed in
+    And The internal user has found the associated respondent
+
+  Scenario: 'To' field will be pre-populated with respondent name (up to 100 characters)
+    When they choose to send them a secure message and navigated to the 'send message' page
```

Comment:
Can we try to be more explicit than they. Respondent or internal user?

Reply:
The wording is copied directly from the story acceptance criteria.